tasks/beats-config.yml

```diff
@@ -85,3 +85,15 @@
     group: root
   when: default_ilm_policy is defined
   notify: restart the service
+
+- name: Assert modules.path presence
+  assert:
+    that:
+      - beat_conf.filebeat.config.modules.path is defined
```

Comment:
I think this should not be specific only to filebeats. Consider changing this to `{{ beat }}` instead.